### PR TITLE
modify existing lessons and add one

### PR DESCRIPTION
**Pull request by JalonSolov**

### CODE REVIEW

File: lessons/2.writing-our-first-program.md [hunk 1]
```diff
@@ -3,8 +3,54 @@ Writing Our First Program
...
-	println('hello world')
+	println('Hello, World!')
 }
 ```
+
+In V, functions are declared with the `fn` keyword, followed by the name of the
+function (in this case `main`), the parameters to the function, if any (in this
+case there aren't any), then an opening curly brace, `{`, on the same line.
+
+V also has 2 ways to output text - one that only prints what is given to it,
```
Comment (larpon) [inline lessons/2.writing-our-first-program.md:19]: If this for absolute beginners maybe mention *where* the output can be seen? (A terminal/shell??)

File: lessons/2.writing-our-first-program.md [hunk 2]
```diff
@@ -3,8 +3,54 @@ Writing Our First Program
...
+`print`, and one that automatically adds a linefeed to the end of whatever it
+is given, `println`.
+
+Using
+```v
+println('Hello, World!')
+```
+above is the equivalent to
+```c
+println('Hello, World!')
```
Comment (larpon) [inline lessons/2.writing-our-first-program.md]: This is not C 🙂

File: lessons/2.writing-our-first-program.md [hunk 3]
```diff
@@ -3,8 +3,54 @@ Writing Our First Program
...
+Finally, the closing curly brace, `}`, to end the function.
+
+One thing to clarify - V doesn't care if you use single quotes (`'`) or double
+quotes (`"`) around a string of characters, the opening and closing quotes just
+have to match.  This means you can't use `'a string"` or `"a string'` but either
+`'a string'` or `"a string"` are fine.
+
+This is unusual amongst programming programming languages, where you must only
+use one or the other (in C, for example, only double quotes are valid around
+strings), or if both types are supported, they have different meanings.  In V,
```
Comment (larpon) [inline lessons/2.writing-our-first-program.md]:
```suggestion
strings), or if both types are supported, they have different meanings. In V,
```

**Review by larpon — CHANGES_REQUESTED** [review]

File: lessons/2.writing-our-first-program.md (hunk 2 above)
Comment (JalonSolov) [reply lessons/2.writing-our-first-program.md]: Oops... copy/paste error.

File: lessons/2.writing-our-first-program.md (hunk 1 above)
Comment (JalonSolov) [reply lessons/2.writing-our-first-program.md:19]: Hmm... that gets tricky, as you may have the output redirected to a file, etc.  Let me see...

**Review by spytheman — APPROVED** [review]
Excellent explanation.

File: lessons/2.writing-our-first-program.md (hunk 1 above)
Comment (spytheman) [reply lessons/2.writing-our-first-program.md:19]: We can also link to https://play.vlang.io/ . It will stay up, unlike others...

**Pull request merged**